Gemfile.lock

```diff
@@ -7,41 +7,73 @@ PATH
 GEM
   remote: https://rubygems.org/
   specs:
-    diff-lcs (1.1.3)
-    ffi (1.1.5)
-    guard (1.4.0)
-      listen (>= 0.4.2)
-      thor (>= 0.14.6)
-    guard-rspec (2.1.0)
-      guard (>= 1.1)
-      rspec (~> 2.11)
-    listen (0.5.3)
-    mock_redis (0.5.4)
-    rake (0.9.2.2)
-    rb-fsevent (0.9.2)
-    rb-inotify (0.8.8)
-      ffi (>= 0.5.0)
-    redis (3.2.0)
-    rspec (2.11.0)
-      rspec-core (~> 2.11.0)
-      rspec-expectations (~> 2.11.0)
-      rspec-mocks (~> 2.11.0)
-    rspec-core (2.11.1)
-    rspec-expectations (2.11.3)
-      diff-lcs (~> 1.1.3)
-    rspec-mocks (2.11.3)
-    thor (0.16.0)
-    timecop (0.5.2)
+    coderay (1.1.3)
+    diff-lcs (1.5.0)
+    ffi (1.15.5)
+    formatador (1.1.0)
+    guard (2.18.0)
+      formatador (>= 0.2.4)
+      listen (>= 2.7, < 4.0)
+      lumberjack (>= 1.0.12, < 2.0)
+      nenv (~> 0.1)
+      notiffany (~> 0.0)
+      pry (>= 0.13.0)
+      shellany (~> 0.0)
+      thor (>= 0.18.1)
+    guard-compat (1.2.1)
+    guard-rspec (4.7.3)
+      guard (~> 2.1)
+      guard-compat (~> 1.1)
+      rspec (>= 2.99.0, < 4.0)
+    listen (3.7.1)
```

Comment:
### [listen](https://github.com/guard/listen)

**Major** version upgrade :chart_with_upwards_trend::exclamation: 0.5.3 → 3.7.1

[_[change-log](https://github.com/guard/listen/releases), [source-code](https://github.com/guard/listen/tree/v3.7.1), [gem-diff](https://my.diffend.io/gems/listen/0.5.3/3.7.1)_]





<details>
<summary>Commits</summary>

A change of **1055** commits. See the full changes on [the compare page](https://github.com/guard/listen/compare/v0.5.3...v3.7.1).

These are the first 10 commits:
- (f72ba63) [avoid gemspec "duplicate dependency" validation error](https://github.com/guard/listen/commit/f72ba630cb601d8099bfb23c5988a9e12b5785b0)
- (0db811a) [Merge pull request #384 from guard/workaround_rubygems_spec_…](https://github.com/guard/listen/commit/0db811a8bc1825b22569ba1c7f05293523a8c1a9)
- (befe13c) [remove unused spec code](https://github.com/guard/listen/commit/befe13c8760cc95131d88e0fbcf9f9d7a1b0008f)
- (66d6079) [allow stopping when not fully initialized](https://github.com/guard/listen/commit/66d607945097ae6e4228e2db8f133124c34b4543)
- (9b2d077) [add TODO](https://github.com/guard/listen/commit/9b2d077ab369fe381ab2ad7d6e6d24be3041f4e5)
- (8368b87) [Merge pull request #383 from guard/fix_state_transition](https://github.com/guard/listen/commit/8368b875a3d6f5829633e23aecb564af187a2a41)
- (1684919) [Release 3.1.2](https://github.com/guard/listen/commit/168491903a9686ad92f87b9147a2e8fdc0cb9f51)
- (03c7052) [activate JRuby-9.0.5.0 on Travis](https://github.com/guard/listen/commit/03c7052f78f9106d82760f09003a7a2ed5451ebe)
- (e86dccf) [avoid unkillable thread problem on JRuby](https://github.com/guard/listen/commit/e86dccf33327477ef67d658a4533313cedb6f594)
- (c23ff1c) [get JRuby to fail with ENOTDIR like MRI does](https://github.com/guard/listen/commit/c23ff1cb2c43b8d6cc715450d68d45fa41b4b792)


</details>